src/framework/_flex.scss

```diff
@@ -0,0 +1,361 @@
+/*============================================================================*\
+   Flex definitions
+\*============================================================================*/
```

Comment:
You should add a local declaration of the `$has-classes` variable at the top of this file.